lib/core/src/sync/model/data.rs

```diff
@@ -170,10 +180,23 @@ pub(crate) struct ReceiveSyncData {
     pub(crate) receiver_amount_sat: u64,
     pub(crate) mrh_address: String,
     pub(crate) created_at: u32,
+    #[serde(default)]
+    pub(crate) expiry_at: Option<u32>,
     pub(crate) payment_hash: Option<String>,
     pub(crate) description: Option<String>,
 }
 
+impl ReceiveSyncData {
+    pub(crate) fn merge(&mut self, other: &Self, updated_fields: &[String]) {
+        for field in updated_fields {
+            match field.as_str() {
+                "expiry_at" => clone_if_set(&mut self.expiry_at, &other.expiry_at),
```

Comment:
should we sync the expiry_at? Asking since it is set both on creation time and when recovering.

Reply:
It's not absolutely necessary